esp/esp/tagdict/models.py

```diff
@@ -57,15 +79,17 @@ def getProgramTag(cls, key, program=None, default=None, ):
         Given a key and program, return the corresponding value as string.
         If the program does not have the tag set, return the global value.
         """
+        if key not in all_program_tags:
+            logger.warning("Tag %s not in list of program tags", key)
```

Comment:
Should this also have a boolean warning?  (If so, you'll need to make sure it doesn't get tripped by `getBooleanTag` calling `getProgramTag`.)